test/schema/me/saved_artworks.js

```diff
@@ -0,0 +1,50 @@
+import { resolve } from "path"
+import { readFileSync } from "fs"
+
+import schema from "../../../schema"
+import { runQuery } from "../../utils"
+
+describe("me { saved_artwork", () => {
+  describe("Handles getting collection metadata", () => {
+    const Collection = schema.__get__("Collection")
+    let gravity = null
+    beforeEach(() => {
+      gravity = sinon.stub()
+      gravity.with = sinon.stub().returns(gravity)
+
+      Collection.__Rewire__("gravity", gravity)
+    })
+
+    afterEach(() => {
+      Collection.__ResetDependency__("gravity")
+    })
+
+    it("returns artworks for a collection", () => {
+      const artworksPath = resolve("test", "fixtures", "gravity", "artworks_array.json")
+      const artworks = JSON.parse(readFileSync(artworksPath, "utf8"))
+      gravity
+        .withArgs("collection/saved-artwork/artworks", { size: 10, offset: 0, total_count: true, user_id: null })
```

Comment:
#nitpicky but `user_id` should def never be null, since this is just a spec mind updating it here to some dummy value? That at least demonstrates the intent better than `null` (IMO).